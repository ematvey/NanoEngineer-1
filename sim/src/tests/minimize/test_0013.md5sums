stdout 990B7471A8601BE7B57A390E2AD3E6EE
stderr D41D8CD98F00B204E9800998ECF8427E
