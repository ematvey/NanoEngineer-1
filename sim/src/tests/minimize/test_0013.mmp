mmpformat 050130 required; 050217 optional
kelvin 300
group (Data)
csys (HomeView) (1.000000, 0.000000, 0.000000, 0.000000) (10.000000) =
(0.000000, 0.000000, 0.000000) (1.000000)
csys (LastView) (0.936981, 0.308166, -0.163947, 0.014883) (5.314410) =
(0.000000, 0.000000, 0.000000) (1.000000)
datum (XY) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (0.000000, =
0.000000, 1.000000) (0.000000, 1.000000, 0.000000)
datum (YZ) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (1.000000, =
0.000000, 0.000000) (0.000000, 1.000000, 0.000000)
datum (ZX) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (0.000000, =
1.000000, 0.000000) (0.000000, 1.000000, 0.000000)
egroup (Data)
group (Untitled)
mol (Chunk-C1) def
atom 1 (1) (0, 0, 0) def
atom 2 (1) (0, 0, 5000) def
bond1 1
egroup (Untitled)
end1
group (Clipboard)
egroup (Clipboard)
end molecular machine part Untitled
