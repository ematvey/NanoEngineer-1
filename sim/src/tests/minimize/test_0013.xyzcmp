2
RMS=0.000216
H 0.000000 0.000000 0.3715
H 0.000000 0.000000 -0.3715
