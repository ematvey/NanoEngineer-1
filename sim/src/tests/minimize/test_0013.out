======= test_0013.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
0
======= stderr =======
== structure comparison ==
